R/metadata.R

```diff
@@ -181,12 +190,14 @@ imeta <- function(
 #' Use SQL-like expressions to query data objects and collections based on different properties.
 #'
 #' @param query GeneralQuery for searching the iCAT database.
-#' @param limit Maximum number of rows to return. Defaults to 100.
-#' @param offset Number of rows to skip for paging. Defaults to 0.
+#' @param limit Maximum number of rows to return. Depracated.
+#' @param offset Number of rows to skip for paging. Depracated.
 #' @param type Type of query: 'general' (the default) or 'specific'.
 #' @param case_sensitive Whether the string matching in the query is case sensitive.
 #'   Defaults to `TRUE`.
 #' @param distinct Whether only distinct rows should be listed. Defaults to `TRUE`.
+#' @param parser Which parser to use: genquery1 or genquery2. Defaults to genquery1.
+#' @param sql_only Whether to use SQL syntax. Defaults to `FALSE`. Needs Genquery2.
```

Comment:
This parameter is the equivalent of 'dry-run'... it requests that the server generate and return to the caller the SQL that *would* have been run.  It does NOT execute the SQL.

`sql_only: An integer which instructs the API plugin to return SQL without executing it.`

From GenQuery2 directly...
https://github.com/irods/irods_api_plugin_genquery2/?#api-plugin

I suggest...

```
Whether to return SQL without executing it. Defaults to `FALSE`. Needs genquery2.
```

Reply:
Did not understand the parameter. Will change this accordingly.